dashboard/src/features/app/components/Container.jsx

```diff
@@ -90,8 +90,6 @@ export default connect(
     authOk: !state.core.requireClientToken || state.core.validToken,
     configKnown: state.core.configKnown,
     configured: state.core.configured,
-    buildCommit: state.core.buildCommit,
-    buildDate: state.core.buildDate,
```

Comment:
Can we remove these from https://github.com/chain/chain/blob/main/dashboard/src/features/app/components/Container.jsx#L93-L94 as well?